docs/apm/getting-started.asciidoc

```diff
@@ -2,21 +2,20 @@
 [[apm-getting-started]]
 == Getting Started
 
-If you have not already installed and configured Elastic APM, the
-**APM > Getting Started** page in {kib} provides information to help you
-complete the setup.
+If you have not already installed and configured Elastic APM,
+the Setup Instructions page in {kib} provides information to help you complete the setup.
 
 [role="screenshot"]
-image::apm/images/apm-setup.jpg[Installation instructions on the APM page in Kibana]
+image::apm/images/apm-setup.png[Installation instructions on the APM page in Kibana]
 
-After you install the Elastic APM agent library in your application, the
-application automatically appears in the APM UI in {kib}. No further
-configuration is required.
+After you install the Elastic APM agent library in your application,
+the application automatically appears in the APM UI in {kib}.
+No further configuration is required.
 
-If you also use Elastic Stack for logging and server-level metrics, you can
-optionally import the APM dashboards that come with the APM Server. You can use
-these APM-specific visualizations to correlate APM data with other data sources.
-To get the dashboards, click the "Load Kibana objects" button at the bottom of the Getting Started guides for APM in Kibana.
+If you also use the Elastic Stack for logging and server-level metrics,
```

Comment:
@formgeist - Sorry, dumb question. I still don't 100% understand what is happening to the dashboards for APM. Also, looking at the _Load Kibana Objects_ text: `Load index pattern, visualizations, and pre-defined dashboards.`, I'm not sure what a "visualization" vs "pre-defined dashboard" is.

Ultimately, my question is: Does this documentation paragraph need to change or go away? If not, can you clarify on the difference between dashboard and visualization?

Reply:
@bmorelli25 So we've shipped the APM server with some pre-defined visualizations and dashboards, mainly for OSS users, that you could use to analyze your data in the Visualize or Dashboards features in Kibana. We're choosing to remove them from the Kibana objects, so when users load the Kibana objects (includes the index pattern etc.), these visualizations and dashboards are no longer available. This is considered a breaking change, and therefore we're waiting until the next major release to remove them. So for now we can keep the copy as is, but we'll have to update it in time for when they're gone. That's also when we'll possibly include a link to the new contrib repo and explain its use.

Reply:
Fantastic. That makes so much more sense. Thank you 🥇

Reply:
No problemo 👍